script/electron-builder.js

```
@@ -0,0 +1,104 @@
+        }
+        if (extraTransformer != null) {
+            return extraTransformer(file);
+        }
+        else {
+            return null;
+        }
+    };
+}
+async function modifyMainPackageJson(file, extraMetadata, isRemovePackageScripts, isRemovePackageKeywords) {
+    const mainPackageData = JSON.parse(await fs.readFile(file, "utf-8"));
+    if (extraMetadata != null) {
+        builder_util_1.deepAssign(mainPackageData, extraMetadata);
+        return JSON.stringify(mainPackageData, null, 2);
+    }
+    return null;
+}
+/// END Monkey-Patch
+
+const builder = require("electron-builder")
+const Platform = builder.Platform
+
+const generate = require('./lib/generate-metadata.js')
+
+let options = {
+  "appId": "link.mauricioszabo.pulsar",
+  "npmRebuild": false,
+  "publish": null,
+  "extraResources": [
+    {
+      "from": "apm",
+      "to": "app/apm"
+    }
+  ],
+  compression: "normal",
+  "linux": {
+    "icon": "resources/app-icons/atom-community.png",
+    "category": "Development",
+    "synopsis": "A hackable text editor for the 22nd century",
+    "target": [
+      {
+        "target": "appimage",
+        "arch": "x64"
+      },
+      {
+        "target": "deb",
+        "arch": "x64"
+      },
+      {
+        "target": "rpm",
+        "arch": "x64"
+      }
+    ]
+  },
+  "mac": {
+    "icon": "resources/app-icons/atom-community.png",
+    "category": "Development"
+  },
+  "win": {
+    "icon": "resources/app-icons/atom-community.ico",
+    "target": [
+      { "target": "nsis" },
+      { "target": "portable" }
+    ]
+  },
+  "extraMetadata": {
+  }
+}
+
+async function main() {
+  const package = await fs.readFile('package.json', "utf-8")
+  options.extraMetadata = generateMetadata(JSON.parse(package))
+  builder.build({
+    config: options
+  }).then((result) => {
+    console.log("Built binaries")
+    fs.mkdir('binaries').catch(() => "")
```

Why do you ignore the error?